src/libraries/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.cs

```diff
@@ -2372,6 +2372,7 @@ public async Task ConnectCallback_ContextHasCorrectProperties_Success(bool syncR
                         else
                         {
                             await s.ConnectAsync(context.DnsEndPoint, token);
+                            await Task.Delay(1);
```

Comment:
Is this just to ensure we definitely go async? If so why not Task.Yield? And maybe a comment?

Reply:
I wanted to make it more likely that it wouldn't have completed by the time we got back out to the entrypoint Send call.  `await Task.Yield()` will ensure that the containing async method will complete asynchronously, but when it's immediate caller checks its IsCompleted, it could have actually completed by then.  I could have chosen a longer delay time, but didn't want to make the test take long too long, so I just settled on a short delay.